src/cpr_sdk/vespa.py

```diff
@@ -36,7 +36,7 @@ def split_document_id(document_id: str) -> tuple[str, str, str]:
     return namespace, schema, data_id
 
 
-def find_vespa_cert_paths() -> tuple[Path, Path]:
+def find_vespa_cert_paths() -> tuple[Union[str, None], Union[str, None]]:
```

Comment:
**suggestion:** Saves defining the `Optional` type manually.

```suggestion
def find_vespa_cert_paths() -> tuple[Optional[str], Optional[str]]:
```

I wonder too if this could be refactored to be a singular `Optional`, unless there's a valid scenario where only 1 of the paths would be `None`.

```suggestion
def find_vespa_cert_paths() -> Optional[tuple[str, str]]:
```